gh-pages/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Daily Builds with Updated Viewers</title>
+</head>
+<body>
+    <a href="https://simularium.github.io/simularium-website/dev">App with viewer at development head</a>
+    <a href="https://simularium.github.io/simularium-website/stable">App with latest released stable viewer</a>
```

Comment:
would relative urls "dev" and "stable" work here?

Reply:
oh yeah they would